.github/workflows/test.yml

```diff
@@ -50,21 +50,14 @@ jobs:
       - run: yarn build:schema
         working-directory: ./app
       - run: git diff --exit-code || (echo 'The contents of the `schema` folder are out of sync with `app/server/schema.graphql`. Did you forget to \"yarn build:schema\"?' && false)
-  db-unit-tests:
+  db-tests:
     needs: install-dev-tools
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
       - run: make db_unit_tests
-  db-style-tests:
-    needs: install-dev-tools
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: dev env setup
-        uses: ./.github/actions/dev-env-setup
       - run: git submodule update --init
```

Comment:
The submodule update could be in the `dev-env-setup` action , since it applies too all the submodules we might have in the future